### ISSUE TITLE
How to fine-tune LVIS dataset?

### ISSUE THREAD

**Issue by mahuixian** [microsoft/GLIP#117]
**When I use the following command for fine-tune, an error will be reported：**
python tools/finetune.py --config-file configs/pretrain/glip_Swin_L.yaml --ft-tasks configs/lvis/minival.yaml --custom_shot_and_epoch_and_general_copy "0_200_1" --evaluate_only_best_on_test --push_both_val_and_test MODEL.WEIGHT MODEL/glip_large_model.pth SOLVER.USE_AMP True TEST.DURING_TRAINING True TEST.IMS_PER_BATCH 4 SOLVER.IMS_PER_BATCH 4 SOLVER.WEIGHT_DECAY 0.05 TEST.EVAL_TASK detection MODEL.BACKBONE.FREEZE_CONV_BODY_AT 2 MODEL.DYHEAD.USE_CHECKPOINT True SOLVER.FIND_UNUSED_PARAMETERS False SOLVER.TEST_WITH_INFERENCE True SOLVER.USE_AUTOSTEP True DATASETS.USE_OVERRIDE_CATEGORY True SOLVER.SEED 10 DATASETS.SHUFFLE_SEED 3 DATASETS.USE_CAPTION_PROMPT True DATASETS.DISABLE_SHUFFLE True

![image](https://github.com/microsoft/GLIP/assets/64054254/f8626bca-14fc-4c24-b023-0aeab39bc3e8)

**How to solve this problem？**
